lib/node_modules/@stdlib/stats/base/dists/bernoulli/entropy/README.md

```diff
@@ -130,6 +130,93 @@ for ( i = 0; i < 10; i++ ) {
 
 <!-- /.references -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dists/bernoulli/entropy.h"
+```
+
+#### stdlib_base_dists_bernoulli_entropy( p )
+
+Returns the [entropy][entropy] of a [Bernoulli][bernoulli-distribution] distribution with success probability `p` (in [nats][nats]).
+
+```c
+double out = stdlib_base_dists_bernoulli_entropy( 0.1 );
+// returns ~0.325
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` minimum support
```

Comment:
You're missing periods in your descriptions.

Reply:
done this